Source/Documentation/Manual/timetable.rst

```diff
@@ -496,6 +505,26 @@ reference is in the form of ``train name:timetable description``, where the
 description is the text at the intersection of the first ``#comment`` row and 
 ``#comment`` column in the other timetable file.
 
+Train Commands
+''''''''''''''
+
+Gradient
+    To provide a realistic behaviour on gradients, the gradient command has been introduced.
+
+    Syntax    
+        ``$gradient /perc=n /speed=s``
+        
+        The gradient command must be set in a ``#speed`` field of a train.
+
+    Functionality    
+        When gradient is set, forced acceleration is turned off if the leading engine of the train is on a 
+        rising gradient which exceeds n percent.
+        However, forced acceleration is turned on if the speed of the train drops below speed s. 
+        This is to ensure the train will not stall, and also that the train will restart if it has come to a stop on a 
+        gradient, e.g. due to a signal. 
+        The dimension of the speed value depends on the definition of the speed field (see manual for details of speed fields).
```

Comment:
Maybe add a link to the relevant part of the manual?

Reply:
Hi Cesar,

Thanks for spotting that.
I hope I've understood what Rob was intending.

Chris